packages/docs/content/components/text-field/label-masked-field.mdx

```diff
@@ -0,0 +1,130 @@
+  storyId="components-text-field--label-masked-currency"
+/>
+
+### Phone number
+
+<StorybookExample
+  componentName="phone"
+  storyId="components-text-field--label-masked-phone"
+/>
+
+### Social Security number
+
+<StorybookExample
+  componentName="ssn"
+  storyId="components-text-field--label-masked-ssn"
+/>
+
+### Zip code
+
+<StorybookExample
+  componentName="zip"
+  storyId="components-text-field--label-masked-postal-code"
+/>
+
+## Code 
+
+### React
+
+Passing a `labelMask` prop into a [`TextField`](/components/text-field/) component with a valid masking function will turn it into a label mask. You can either import one of the named, built-in mask functions (`SSN_MASK`, `ZIP_MASK`, `PHONE_MASK`, or `CURRENCY_MASK`), or you can provide a custom mask function. The following table shows the `TextField` prop that is specific to label masks:
+
+<PropTable componentName="TextField" subset={["labelMask"]} />
+
+#### Handling input changes
+
+When tracking state in a parent component, treat the value as a raw input string, which is updated by the `onChange` handler. Do not try to unmask or clean the value before passing it back to the `TextField` component instance. The intention with this is to keep the API and behavior of the component as simple as possible. You don't need to handle `onBlur` separately from `onChange`. You only need to update your value when `onChange` is called, and it will be called whenever the value of the input changes, including on blur. A more appropriate place to clean the input would be in your validation code.
+
+```js
+// A simplified example of change handling. This is not a recommendation
+// for how to structure error messages or validation code but serves only
+// to illustrate how to set and get values from the label-masked field.
+
+import { useState } from "react";
+import { Button, TextField, SSN_MASK } from '@cmsgov/design-system';
+
+export const MyForm = () => {
+  const [value, setValue] = useState('');
+  const [error, setError] = useState();
+
+  function validate() {
+    if (!isValidSsn(value)) {
+      e.preventDefault();
+      setError('You have entered an invalid SSN.');
+    }
+  }
+
+  return (
+    <form onSubmit={validate}>
+      {error && <Alert variation="error">{error}<Alert>}
+      <TextField
+        label="Social security number (SSN)"
+        labelMask={SSN_MASK}
+        name="ssn_example"
+        value={value}
+        onChange={setValue}
+      />
+      <Button type="submit">Submit</Button>
+    </form>
+  )
+};
+```
+
+### Styles
+
+## Guidance
+
+### When to use
+
+- In fields with a specific expected format like Social Security Number or ZIP code, a label mask allows you to constrain and shape the information being entered without impairing the user's ability to copy/paste or correct mistyping
+
+### When to consider alternatives
+
+- When the input requires a free-form field. When there isn't a common input pattern to use, masking is not appropriate.
```

Comment:
Should this be 2 bullet points? 

> - When the input requires a free-form field. 
> - When there isn't a common input pattern to use, masking is not appropriate.

Reply:
This is the same item in USWDS where the second sentence is explaining the first in more detail. This could use some finessing though.

Reply:
I've combined the sentences in my update, but it's still not great.